src/epics/revisions.js

```diff
@@ -19,11 +19,22 @@ const buildUrl = ( state ) => {
 	return url;
 };
 
-const fetchRevisions = ( action$, store ) => (
+export const hideLoader = ( action$ ) => (
+	action$
+		.ofType( 'PAGES_UPDATE' )
+		.flatMap( () => Observable.of( RevisionsActions.setStatusReady() ) )
+);
```

Comment:
@dayllanmaza 
Since this is synchronous, you can just do this in src/reducers/revisions/status.js (i.e. add `PAGES_UPDATE` to your switch statement).

Reply:
Done